src/menus/FileMenus.cpp

```diff
@@ -336,6 +336,14 @@ void OnExportLabels(const CommandContext &context)
 
    LabelFormat format = LabelTrack::FormatForFileName(fName);
 
+   // Ensure filename has correct extension for selected format
+   if (format == LabelFormat::PODCAST_CHAPTERS_JSON && fName.Right(5).CmpNoCase(wxT(".json")) != 0) {
+      // Add .json extension if not present
```

Comment:
The comment here does not match what the code does, it overrides the extension if it does not match ".json", do we really want to restrict the user from exporting the file with an alternative extension?